app/controllers/EditionsController.scala

```diff
@@ -98,4 +103,14 @@ class EditionsController(db: EditionsDB,
       NoContent
     }.getOrElse(NotFound(s"Issue $id not found"))
   }
+
+  def getPrefillForCollection(id: String) = AccessAPIAuthAction.async { req =>
+    db.getCollectionPrefillQueryString(id).map { prefillUpdate =>
+      capi.getPrefillArticles(prefillUpdate.issueDate, prefillUpdate.prefill, prefillUpdate.currentPageCodes).map { body =>
+        val json = "{\"response\":" + body.asJson.noSpaces + "}"
```

Comment:
why do we need to do this?

Reply:
The CAPI client and the CAPI API return different objects :(